packages/autocertifier-client/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@streamr/autocertifier-client",
+  "version": "100.0.0-pretestnet.0",
+  "description": "Autocertifier Client for Streamr Network",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/streamr-dev/network.git",
+    "directory": "packages/autocertifier-client"
+  },
+  "main": "dist/src/exports.js",
+  "types": "dist/src/exports.d.ts",
+  "license": "STREAMR NETWORK OPEN SOURCE LICENSE",
+  "author": "Streamr Network AG <contact@streamr.network>",
+  "scripts": {
+    "generate-protoc-code": "./proto.sh",
+    "build": "tsc -b tsconfig.node.json",
+    "check": "tsc -p ./tsconfig.jest.json --noEmit",
+    "clean": "jest --clearCache || true; rm -rf dist *.tsbuildinfo node_modules/.cache || true",
+    "eslint": "eslint --cache --cache-location=node_modules/.cache/.eslintcache/ '*/**/*.{js,ts}'",
+    "prepublishOnly": "npm run clean && NODE_ENV=production tsc -b tsconfig.node.json"
+  },
+  "dependencies": {
+    "@protobuf-ts/runtime-rpc": "^2.8.2",
+    "@streamr/utils": "100.0.0-pretestnet.0",
+    "eventemitter3": "^5.0.0",
+    "node-forge": "^1.3.1",
+    "request": "^2.88.2"
```

Comment:
This package has been deprecated https://www.npmjs.com/package/request. Let's use e.g. `node-fetch` instead or add an entry to future improvement.

Reply:
already has todo comment